### PR TITLE
[16.0][MIG] repair_purchase_return: Migration to 16.0

### PR DESCRIPTION
**Pull request by PauBForgeFlow**

### CODE REVIEW

File: repair_purchase_return/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The purpose of this module is to modify the behaviour of the repair standard module. Achieving more flexibility in a repair process.
+
+The first change that it applies is the modification of the creation of stock moves in the repair flow:
+
+* Confirm Repair: creates draft stock moves on confirmation of the repair order.
+* Start Repair: confirms the stock moves.
+* End Repair: completes the stock moves and ends the repair order.
+
+Another feature which this module includes is that it allows to add components during all the different stages of the repair process, which adjusts better to the needs of some businesses.
```
Comment (mariadforgeflow) [inline repair_purchase_return/readme/DESCRIPTION.rst]: This Description, as well as the Usage, don't match with the module's functionality (they are from repair_stock_move module). Could you please update them?

Comment (PauBForgeFlow) [reply repair_purchase_return/readme/DESCRIPTION.rst]: Updated

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Comment by AaronHForgeFlow** [comment]
Rebased to rebuild runboat, sorry

**Comment by LoisRForgeFlow** [comment]
@AaronHForgeFlow @PauBForgeFlow  what is missing for this one?

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
/ocabot merge nobump

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+odoo-addon-purchase_return@git+https://github.com/OCA/purchase-workflow.git@refs/pull/1975/head#subdirectory=setup/purchase_return
```
Comment (LoisRForgeFlow) [inline test-requirements.txt]: Aborting merge, @JordiBForgeFlow could you remove this?

Comment (JordiBForgeFlow) [reply test-requirements.txt]: @LoisRForgeFlow Done!

**Pull request merged**